BepInEx.Core/Console/Windows/WindowsConsoleDriver.cs

```diff
@@ -143,7 +144,7 @@ public void SetConsoleColor(ConsoleColor color)
 
     private static Stream OpenFileStream(IntPtr handle)
     {
-        if (ReflectionHelper.IsCore)
+        if (PlatformTriple.Current.Runtime.Target == RuntimeKind.CoreCLR)
```

Comment:
This can be more simply retrieved via `PlatformDetection.Runtime`

Reply:
done.